scripts/gain_scheduler_node.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
```

Comment:
Can this file be moved to `buoy_examples` or does it need to be in `scripts`?

Reply:
@hamilton8415 and I have been pondering if the gain scheduler really serves as a good example or if we should just leave it in our internal repo.

If we do leave it here and test it on the buoy (should be tested soon), I just remembered I should make some rosdeps and/or a requirements.txt for it (there are a few python packages that are needed).

Reply:
Got it. I'll leave it up to you if you want to add that as part of this PR or in a follow up.